scripts/db_migrator.py

```diff
@@ -717,9 +721,36 @@ def version_3_0_4(self):
 
     def version_3_0_5(self):
         """
-        Current latest version. Nothing to do here.
+        Version 3_0_5
         """
         log.log_info('Handling version_3_0_5')
+        # Removing LOGLEVEL DB and moving it's content to CONFIG DB
+        # Removing Jinja2_cache
+        warmreboot_state = self.stateDB.get(self.stateDB.STATE_DB, 'WARM_RESTART_ENABLE_TABLE|system', 'enable')
```

Comment:
Please verify if in current implementation of fast-reboot LOGLEVEL_DB is being flushed, if it is not flushed as part of the reboot this scenario should be handled as well.
However in fast-reboot over warm-reboot implementation fast-reboot sets WARM_RESTART_ENABLE_TABLE|system:enable=true so the fast-reboot scenario is covered by this logic as well.
I think it should be mentioned that this PR should be merged after fast-reboot over warm-reboot PRs are being merged as it depends on them for fast-reboot scenario (unless loglevel db is being flushed of course)